bigquery_etl/docs/validate_docs.py

```diff
@@ -27,35 +27,6 @@
 standard_args.add_log_level(parser)
 
 
-def sql_for_dry_run(file, parsed_routines, project_dir):
-    """
-    Return the example SQL used for the dry run.
-
-    Injects all UDFs the example depends on as temporary functions.
-    """
-    dry_run_sql = ""
-
-    with open(file) as sql:
-        example_sql = sql.read()
-
-        # add UDFs that example depends on as temporary functions
-        for udf, raw_routine in parsed_routines.items():
-            if udf in example_sql:
-                query = "".join(raw_routine.definitions)
-                dry_run_sql += sub_local_routines(query, project_dir, parsed_routines)
-
-        dry_run_sql += example_sql
-
-        for udf, _ in parsed_routines.items():
-            # temporary UDFs cannot contain dots, rename UDFS
-            dry_run_sql = dry_run_sql.replace(udf, udf.replace(".", "_"))
-
-        # remove explicit project references
-        dry_run_sql = dry_run_sql.replace(os.path.basename(project_dir) + ".", "")
-
-    return dry_run_sql
-
-
```

Comment:
`sub_local_routines` already handles recursive references and transforming `dataset.func` and `project.dataset.func` to `datataset_func`, so I replaced this function with a call directly to `sub_local_routines`